doc/ledger-reward-events.md

```diff
@@ -0,0 +1,47 @@
+# Ledger Reward Events
+
+This is an ambit claim of what data the ledger events should provide from the point of view of
+`db-sync`.
+
+## Prologue
+
+In the context of `db-sync` and the `ledger`, the word "reward" is slightly misleading and should
+not be narrowly defined as only being payments of staking rewards, but defined much more broadly as
+any and all payments to a stake address. None of these payments are recorded on the block chain as
+part of regular transactions.
+
+Payments to stake addresses are made for one of following reasons:
+* Staking rewards for being a member of a stake pool.
+* Staking rewards for being an owner of a stake pool.
+* MIR payments from the treasury.
+* MIR payments from the reserves.
+* Refunds of the stake pool deposit payment when a pool is de-registered.
+
+Currently, the ledger provides the following reward related events:
```

Comment:
It's probably worth noting that the list below is the name of the wrapped events from the Cardano API. The corresponding ledger events are currently nested according to the ledger rule hierarchy (though we may change this is the near future, a clean up is on order)
* `ShelleyLedgerEventTICK . NewEpochEvent . DeltaRewardEvent . RupdEvent`
* `ShelleyLedgerEventTICK . NewEpochEvent . MirEvent`
* `ShelleyLedgerEventTICK . NewEpochEvent . EpochEvent . PoolReapEvent`
* `ShelleyLedgerEventTICK . NewEpochEvent . TotalRewardEvent`